src/cluster.c

```diff
@@ -2596,9 +2596,9 @@ int clusterProcessPacket(clusterLink *link) {
         resetManualFailover();
         server.cluster->mf_end = now + CLUSTER_MF_TIMEOUT;
         server.cluster->mf_slave = sender;
-        pauseClients(PAUSE_DURING_FAILOVER,
-                     now + (CLUSTER_MF_TIMEOUT * CLUSTER_MF_PAUSE_MULT),
-                     CLIENT_PAUSE_WRITE);
+        pauseServices(PAUSE_DURING_FAILOVER,
+                      now + (CLUSTER_MF_TIMEOUT * CLUSTER_MF_PAUSE_MULT),
+                      PAUSE_SVC_CLIENT_WRITE|PAUSE_SVC_EXPIRE|PAUSE_SVC_EVICT|PAUSE_SVC_REPLICA);
```

Comment:
I was going to suggest `pauseEvents`. Services is definitely weird. Effects is a little clearer, but client processing isn't an effect. In a sense they are tasks, but I think event is more of a term Redis uses. I don't have the benefit of keeping the alignment though, but I don't care about that.

Reply:
Looking at all the options presented so far, i think `pauseActions` is the best.
i'm not sure we have to have a short name for it, it could just be `PAUSE_ACTION_CLIENT_WRITE`

Reply:
The name of the child shall be then `pauseActions`. Thanks.